dbscripts/xml/upgrade/3.1.0_update.xml

```diff
@@ -69,5 +69,11 @@
 		<!-- remove old, removed journal settings migrated/concatenated to the new, existing -->
 		<query>DELETE FROM journal_settings WHERE setting_name IN ('focusScopeDesc', 'reviewPolicy', 'pubFreqPolicy', 'openAccessPolicy', 'customAboutItems', 'contributorNote', 'contributors', 'sponsorNote', 'sponsors', 'history')</query>
 	</sql>
+	<sql><!-- pkp/pkp-lib#2537 -->
+		<!-- copy article_galley_settings into submission_galley_settings -->
+		<query>INSERT INTO submission_galley_settings (galley_id, locale, setting_name, setting_value, setting_type) SELECT galley_id, locale, setting_name, setting_value, setting_type FROM article_galley_settings WHERE setting_name &lt;&gt; 'excludeDoi' AND setting_name &lt;&gt; 'excludeURN'</query>
```

Comment:
It looks like this setting is moved from article-specific to galley-specific; I guess that's intentional? Is there no more role for those settings on the article level? (If it's something we might re-add to articles later, in addition to the galley level, it might be worth leaving the `submission_settings` entries around.)